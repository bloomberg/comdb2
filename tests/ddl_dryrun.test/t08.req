drop table if exists t08
create table t08 { tag ondisk {int i int j} tag "t1" {int i} }$$
dryrun alter table t08 { tag ondisk {int i int j} tag "t1" {int i int j} }
dryrun alter table t08 { tag ondisk {int i int j} }
