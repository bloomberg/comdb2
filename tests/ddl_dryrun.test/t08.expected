(Message='Schema change plan:-')
(Message='    No changes to main data file')
(Message='    Tags are modified')
(Message='    Schema change does not require a table scan ')
(Message='There is no change in the schema')
(Message='Table is already odh enabled.')
(Message='Table records are compressed.')
(Message='Table blobs are compressed.')
(Message='Table already supports in-place updates.')
(Message='Table already supports instant schema change.')
(Message='Live schema change running in parallel scan mode')
(Message='Schema change plan:-')
(Message='    No changes to main data file')
(Message='    Tags are modified')
(Message='    Schema change does not require a table scan ')
(Message='There is no change in the schema')
(Message='Table is already odh enabled.')
(Message='Table records are compressed.')
(Message='Table blobs are compressed.')
(Message='Table already supports in-place updates.')
(Message='Table already supports instant schema change.')
(Message='Live schema change running in parallel scan mode')
